Makefile

```diff
@@ -180,6 +180,7 @@ destroy:  ## Destroy the environment
 		fqdn=`jq -r '.tsb_fqdn' terraform.tfvars.json`; \
 		address=`jq -r "if .ingress_ip.value != \"\" then .ingress_ip.value else .ingress_hostname.value end" outputs/terraform_outputs/terraform-tsb-mp.json`; \
 		cd "tsb/fqdn/$$cloud"; \
+		terraform init ; \
```

Comment:
the state should exist for destroy... not sure why init is required here.

Reply:
what about this one? @boeboe

Reply:
I removed it in my next commit. The reason I added is because I got `module not found` errors while testing this for all clouds with `make k8s` and and `make destroy`. I just realized that the fqdn part is ran at later stages of the `make tsb` flow.